server/models/handlers/signup.js

```diff
@@ -0,0 +1,121 @@
+import java.util.ArrayList;
+ import java.util.Scanner;
+
+
+ public class signup {
+
+        public static void  main(String[] args) {
+           
+           //Create arrays to store the credentials provided by the user 
+        //Array for users who are students
+        ArrayList<userType> users = new ArrayList<userType>(); 
+        //Array for users who are serviceStaff
+        ArrayList<userType> serviceStaff = new ArrayList<userType>(); 
+            
+            //Create a scanner to injest the credentials which the user inputs
+
+            Scanner input = new Scanner(System.in); 
```

Comment:
Not sure why scanner is being used here. The user input is handled by the front end through the sign in form. The endpoint would only need to take in those values to perform queries.

Reply:
Hi Amy, could you point me to some resources I can reference on inputting parameters from the front end sign up form?

Reply:
Would the user need to select a password and input that through the sign up form?

Reply:
im assumming so, since password is needed for a user account, both for signing up and logging in to the system